### DISCUSSION TITLE
Welcome to international_results Discussions!

### DISCUSSION THREAD

**Discussion by martj42** [martj42/international_results#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NextGenSeek** [comment]
Hi, I just wanted to say thank you! 

At every big tournament, a group of friends and I have a score-prediction competition (just for fun and honour). This year, I wanted to enter with AI-predicted scores. I thought it would be a problem to collect all the data, but then I found your dataset, which was very useful for training my model. Thank you.
